packages/db/.env

```
@@ -1,4 +1,7 @@
-DATABASE_URL=postgresql://neondb_owner:npg_fVHg3Ywuax8X@ep-old-snow-ago9cksb-pooler.c-2.eu-central-1.aws.neon.tech/neondb?sslmode=require&channel_binding=require
+# Local Docker PostgreSQL (switch to Neon by changing these values)
+DATABASE_URL=postgresql://skriuw:skriuw_dev@localhost:5432/skriuw
+DATABASE_PROVIDER=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_PROVIDER key should go before the DATABASE_URL key

(UnorderedKey)